psalm.xml.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2021 Nextcloud GmbH and Nextcloud contributors
```

Comment:
Should actually work in the file for this? Otherwise it should be in the reuse.toml to avoid shipping

Reply:
Fixed via https://github.com/nextcloud/twofactor_nextcloud_notification/pull/849/commits/f61e2ebb8d6926e9a61488fa763c673518d340db 👍

Reply:
And psalm still works 🎉
https://github.com/nextcloud/twofactor_nextcloud_notification/actions/runs/10226927204/job/28297794449#step:6:1